custom_components/tuya_local/devices/inkbird_ibt_26s_thermometer.yaml

```diff
@@ -0,0 +1,342 @@
+name: BBQ thermometer
+products:
+  - id: h8qbni4ouwn5gjh0
+    name: Inkbird IBT-26S
+primary_entity:
+  entity: sensor
+  name: Temperature Probe 1
+  class: temperature
+  dps:
+    - id: 107
+      type: integer
+      name: sensor
+      class: measurement
+      mapping:
+        - scale: 100
+      unit: F
+secondary_entities:
+  - entity: sensor
+    name: Temperature Probe 2
+    class: temperature
+    dps:
+      - id: 108
+        type: integer
+        name: sensor
+        class: measurement
+        optional: true
+        mapping:
+          - scale: 100
+        unit: F
+  - entity: sensor
+    name: Temperature Probe 3
+    class: temperature
+    dps:
+      - id: 109
+        type: integer
+        name: sensor
+        class: measurement
+        optional: true
+        mapping:
+          - scale: 100
+        unit: F
+  - entity: sensor
+    name: Temperature Probe 4
+    class: temperature
+    dps:
+      - id: 110
+        type: integer
+        name: sensor
+        class: measurement
+        optional: true
+        mapping:
+          - scale: 100
+        unit: F
+  - entity: sensor
+    name: Temperature Probe 5
+    class: temperature
+    dps:
+      - id: 124
+        type: integer
+        name: sensor
+        class: measurement
+        optional: true
+        mapping:
+          - scale: 100
+        unit: F
+  - entity: sensor
+    name: Temperature Probe Oven
+    class: temperature
+    dps:
+      - id: 125
+        type: integer
+        name: sensor
+        class: measurement
+        optional: true
+        mapping:
+          - scale: 100
+        unit: F
+  - entity: sensor
+    name: Internal temperature
+    class: temperature
+    category: diagnostic
+    dps:
+      - id: 112
+        type: integer
+        name: sensor
+        unit: C
+        class: measurement
+  - entity: select
+    translation_key: temperature_unit
+    category: config
+    dps:
+      - id: 19
+        type: string
+        name: option
+        mapping:
+          - dps_val: F
+            value: fahrenheit
+          - dps_val: C
+            value: celsius
+  - entity: sensor
+    name: Battery level
+    class: battery
+    category: diagnostic
+    dps:
+      - id: 101
+        type: integer
+        name: sensor
+        unit: "%"
+  - entity: binary_sensor
+    name: Battery state
```

Comment:
You do not need the name, if the class is set and unique  Leaving out the name allows HA provided translations to be used.